README.md

```diff
@@ -1 +1,106 @@
-# cap-tech-page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Cap stone project - 
+
+> This is my first capstone project created with html, css, javascript and photoshop
+
+## Recent Fixes
+
+- 
+
+## Built With
+
+- HTML
+  - Use semantic HTML tags.
+  - Apply best practices in HTML code.
+
+- CSS
+  - Use CSS selectors correctly.
+  - Use CSS box model.
+  - Use Flexbox to place elements in the page.
+  - Use grid to place elements in the page.
+  - Media queries use to create UIs adaptable to different screen sizes.
+    - Mobile Max-width: 768px
+    - Desktop Min-width: 768px
+
+- JavaScript
+  - Apply JavaScript best practices and language style guides in code.
+  - Use JavaScript to manipulate DOM elements.
+  - Use JavaScript events.
+  - Use objects to store and access data.
+
+- GitHub
+  - Use GitHub Pages to deploy web pages.
+
+## Live Demo
+
+- [Live link](https://https://187jjay187.github.io/cap-tech-page/)
```

Comment:
- [ ] The link to your live demo is broken. Please fix it and make it available in order to make it work as expected :smile: 

![Screenshot from 2022-10-28 19-13-20](https://user-images.githubusercontent.com/6615394/198714872-8f6f0515-1bf7-4a32-9d73-b73b93c2a426.png)